app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -428,7 +427,7 @@ public boolean onBackPressed() {
             SessionManager.getInstance().removeCurrentSession();
         }
 
-        TelemetryWrapper.controllerBackEvent(ControllerBackLocation.BROWSER);
+        TelemetryWrapper.browserBackControllerEvent();
```

Comment:
I think we don't care about navigating "back" out of the browser app so I would put this in the "canGoBack" fork. But, also doesn't matter TOO much because worst case this will throw off telemetry by 1.